.github/workflows/ci.yaml

```diff
@@ -0,0 +1,65 @@
+name: For each commit and PR
+on:
+  push:
+  pull_request:
+
+jobs:
+  validation:
+    runs-on: ubuntu-latest
+    env:
+      CGO_ENABLED: 0
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '1.14.6'
+    - name: go fmt
+      run: go get golang.org/x/tools/cmd/goimports; goimports -d . | (! grep .)
+    - name: go vet
+      run: go vet ./...
+    - name: golangci-lint
+      uses: golangci/golangci-lint-action@v1
+      with:
+        version: v1.30
+        args: --skip-dirs cmd/packetflow
+    - name: go test
+      run: go test -v ./...    
+    - name: go test coverage
+      run: go test -coverprofile=coverage.txt ./...
+    - name: upload codecov
+      run: bash <(curl -s https://codecov.io/bash)
+    - name: Build binaries
+      run: go build
```

Comment:
Hmm is there an alpine builder? This is going to change hegel from building in alpine/musl to building in ubuntu/glibc. This is probably fine though since I don't think we have anything that will accidentally do a non-static build.